playground/tools/scrap.py

```
@@ -0,0 +1,357 @@
+        "success": True,
+        "keyword": keyword,
+        "total_found": len(enhanced_products),
+        "products": enhanced_products
+    }      
+
+# =============================================================================
+# 🔧 5. 병렬 데이터 수집 도구
+# =============================================================================
+
+@tool
+def enrich_products_with_details(products: List[Dict[str, Any]], max_workers: int = 5) -> List[Dict[str, Any]]:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)